mercury_engine_data_structures/formats/game_model_root.py

```diff
@@ -4,14 +4,14 @@
 from mercury_engine_data_structures.formats.property_enum import PropertyEnum
 
 
-def create(name: str, magic_number: int):
+def create(name: str, magic_number: int, root=gameeditor_CGameModelRoot):
     return construct.Struct(
-        _magic=construct.Const(name, PropertyEnum),
-        _magic_number=construct.Const(magic_number, construct.Hex(construct.Int32ul)),
+        magic=construct.Const(name, PropertyEnum),
+        magic_number=construct.Const(magic_number, construct.Hex(construct.Int32ul)),
```

Comment:
sorry I was looking at an old branch, you can revert this

Reply:
Fixed